backend/app/service/image.go

```diff
@@ -439,6 +439,9 @@ func (u *ImageService) ImageRemove(req dto.BatchDelete) error {
 				}
 				return buserr.WithDetail(constant.ErrInUsed, id, nil)
 			}
+			if strings.Contains(err.Error(), "image has dependent") {
+				return buserr.New(constant.ErrObjectBeDependent)
+			}
 			return err
 		}
 	}
```

Comment:
The added conditional statement checks if the error message contains "image has dependent" and returns `constant.ErrObjectBeDependent` if true. This appears to be an optimized solution to check for specific conditions that can cause errors during image removal, improving efficiency by reducing redundant checks elsewhere in the function.

If you have more changes or need further assistance, feel free to ask!